Editor/GameLiftClientSettingsEditor.cs

```diff
@@ -10,33 +10,35 @@ namespace AmazonGameLift.Editor
     [CustomEditor(typeof(GameLiftClientSettings))]
     public sealed class GameLiftClientSettingsEditor : UnityEditor.Editor
     {
-        private SerializedProperty _isLocalTest;
-        private SerializedProperty _localPort;
+        
         private SerializedProperty _remoteUrl;
         private SerializedProperty _region;
         private SerializedProperty _poolClientId;
+        private SerializedProperty _isAnywhereTest;
+        private SerializedProperty _computeName;
+        private SerializedProperty _fleetID;
+        private SerializedProperty _fleetLocation;
+        private SerializedProperty _profileName;
 
-        private void OnEnable()
-        {
-            _isLocalTest = serializedObject.FindProperty(nameof(GameLiftClientSettings.IsLocalTest));
-            _localPort = serializedObject.FindProperty(nameof(GameLiftClientSettings.LocalPort));
-            _remoteUrl = serializedObject.FindProperty(nameof(GameLiftClientSettings.ApiGatewayUrl));
-            _region = serializedObject.FindProperty(nameof(GameLiftClientSettings.AwsRegion));
-            _poolClientId = serializedObject.FindProperty(nameof(GameLiftClientSettings.UserPoolClientId));
-        }
 
         public override void OnInspectorGUI()
         {
             serializedObject.Update();
             var targetSettings = (GameLiftClientSettings)target;
 
-            EditorGUILayout.PropertyField(_isLocalTest, new GUIContent("Local Testing Mode", "Enabling this to make sample game client connect to game server running on localhost"));
-
+            EditorGUILayout.PropertyField(_isAnywhereTest, new GUIContent("GameLift Anywhere", "GameLift Anywhere Mode"));
+            EditorGUILayout.PropertyField(_region, new GUIContent("AWS Region", "AWS region used for communicating with Cognito and API Gateway"));
+            
+            if (string.IsNullOrWhiteSpace(targetSettings.AwsRegion))
```

Comment:
As discussed, remove AwsRegion (unless it's used for things other than generating websocket URL).